### PR TITLE
chore/update npm log parsing

### PR DESCRIPTION
**Pull request by thezackm**
updating NPM and Logs docs to reflect new built-in rule for parsing `Ktranslate container health` logs automatically. This PR also adds in the missing rule for `HAProxy` logs.

### CODE REVIEW

File: src/content/docs/logs/ui-data/parsing.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ Parsing is the process of splitting unstructured log data into attributes (key/v
 
 To get started with parsing, you may want to watch the following video tutorial on YouTube (approx. 4-1/2 minutes).
 
-<Video id="PD7zGXqS0P0" type="youtube" width="560px" />
+//<Video id="PD7zGXqS0P0" type="youtube" width="560px" />
```
Comment (bradleycamacho) [inline src/content/docs/logs/ui-data/parsing.mdx:18]: Not sure why these forward slashes were added. Are you trying to comment them out @thezackm ?

**Review by bradleycamacho — CHANGES_REQUESTED** [review]
Hey there @thezackm this looks great! Just one small comment and then we can get this approved:

File: src/content/docs/logs/ui-data/parsing.mdx (hunk 1 above)
Comment (thezackm) [reply src/content/docs/logs/ui-data/parsing.mdx]: ugh, nope! I commented those out to get my markdown preview working is VS Code and forgot to remove them 😬 

fixing now!

**Review by paperclypse — APPROVED** [review]
Looks good to me!

**Comment by paperclypse** [comment]
@bradleycamacho I'm merging this in, because @thezackm fixed the issue you raised.

**Pull request merged**